Bluetooth: controller: llcp: lll_scan_aux does not compile with new LLCP

**Describe the bug**
Compilation fails for a sample that includes the file subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c when the new LLCP is selected

**To Reproduce**
- Select the new LLCP (by selecting it with 'west build -t menuconfig' in for example the shell sample (tests/bluetooth/shell)
- build the sample using 'west build'
- compiler errors will show up


**Expected behavior**
The sample must compile 

**Impact**
Showstopper for several samples

**Logs and console output**
Error message is:
/home/ank1/Zephyr/zephyrproject/zephyr/subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c: In function 'isr_rx_connect_rsp':
/home/ank1/Zephyr/zephyrproject/zephyr/subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c:1423:12: error: 'struct lll_conn' has no member named 'max_tx_time'; did you mean 'phy_tx_time'?
 1423 |  conn_lll->max_tx_time = MAX(conn_lll->max_tx_time,

**Environment (please complete the following information):**
N/A

**Additional context**
N/A

--- a/subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c
+++ b/subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c
@@ -1420,12 +1420,21 @@ static void isr_rx_connect_rsp(void *param)
 	struct lll_conn *conn_lll = lll->conn;
 
 #if defined(CONFIG_BT_CTLR_DATA_LENGTH)
+#if defined(CONFIG_BT_LL_SW_LLCP_LEGACY)
 	conn_lll->max_tx_time = MAX(conn_lll->max_tx_time,
 				    PDU_DC_MAX_US(PDU_DC_PAYLOAD_SIZE_MIN,
 						  lll_aux->phy));
 	conn_lll->max_rx_time = MAX(conn_lll->max_rx_time,
 				    PDU_DC_MAX_US(PDU_DC_PAYLOAD_SIZE_MIN,
 						  lll_aux->phy));
+#else
+	conn_lll->dle.eff.max_tx_time = MAX(conn_lll->dle.eff.max_tx_time,
+					    PDU_DC_MAX_US(PDU_DC_PAYLOAD_SIZE_MIN,
+							  lll_aux->phy));
+	conn_lll->dle.eff.max_rx_time = MAX(conn_lll->dle.eff.max_rx_time,
+					    PDU_DC_MAX_US(PDU_DC_PAYLOAD_SIZE_MIN,
+							  lll_aux->phy));
+#endif /* CONFIG_BT_LL_SW_LLCP_LEGACY */
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH*/
 
 	conn_lll->phy_tx = lll_aux->phy;

